src/main/kotlin/racingcar/car/CarFactory.kt

```diff
@@ -1,5 +1,5 @@
 package racingcar.car
 
 object CarFactory {
-    fun getCars(carsCnt: Int): List<Car> = List(carsCnt) { Car(AMGEngine()) }
+    fun getCars(carNames: List<String>): List<Car> = carNames.map { Car(it) }.toList()
```

Comment:
```suggestion
    fun getCars(carNames: List<String>): List<Car> = carNames.map { Car(it) }
```
대체로 collection 관련 함수들(map, filter 등)은 새 collection을 만들어서 반환합니다.
해서, toList 함수를 호출하실 필요가 없습니다. 😄  (이하 관련 내용 동일)